gridstatus/ercot.py

```diff
@@ -2142,6 +2145,32 @@ def _handle_price_corrections(self, docs, verbose=False):
 
         return df
 
+    @support_date_range("HOUR_START")
+    def get_system_wide_actual_load(self, date, end=None, verbose=False):
+        """Get 15-minute system-wide actual load.
+
+        This report is posted every hour and includes only system-wide actual load.
+
+        Args:
+            date (str, datetime): date to get data for
+            end (str, datetime, optional): end time to get data for. Defaults to None.
+            verbose (bool, optional): print verbose output. Defaults to False.
+
+        Returns:
+            pandas.DataFrame: A DataFrame with system actuals data
+        """
+        doc = self._get_document(
+            report_type_id=SYSTEM_WIDE_ACTUALS_RTID,
+            published_before=date,
```

Comment:
i dont think this this gets the time the user would expect. 

`@support_date_range("HOUR_START")` provides a time at the beginning of the hour. 

however, the report published for a particular hour gets published 5 minutes after the end. I think we need to do something like 

```
doc = self._get_document(
            report_type_id=SYSTEM_WIDE_ACTUALS_RTID,
            published_after=date + pd.Timedelta(hours=1), 
            published_before=date + pd.Timedelta(hours=2),
            extension="csv",
            verbose=verbose,
        )
```